README.md

```diff
@@ -388,6 +389,25 @@ docker run -d -v /var/artipie:/var/artipie` -p 80:80 \
   artipie/artipie:latest
 ```
 
+### Credentials settings
+
+Authentication cal also be configured with yaml credentials file:
+
+```yaml
+credentials: 
+  jane: 
+    pass: "plain:qwerty"
+  joe: 
+    login: joe@mail.com
```

Comment:
@olenagerasimova why not use directly:
```yaml
credentials:
  joe@mail.com:
    pass: "sha256:xxxxxxxxxxxxxxxxxxxxxx"
```

Like that there are zero ambiguity, with the current proposal there are two problems:
1. the `joe` key is useless
2. you can have two users with same login even using two different key.

Reply:
@victornoel hm, in my understanding `joe` is not useless, because:
1. I will have to return it as user from `YamlAuth` implementation of `Authentication`, check #157
2. `joe` (not the alternative login) can be used in repository permissions settings as described in #69 and #71

Your second is absolutely fair, I think we should ask @g4s8 about this situation.

Reply:
@victornoel @olenagerasimova yes, `joe` is like a user id, but `joe@mail.com` is a part of login credentials. I think it's ok that user may have multiple credentials, since it's possible that different repository authentications will require different credentials format. E.g. Maven requires HTTP basic, so we can use `joe` and password to authenticate it, but another repository may required different authentication method (e.g. using token), but we still want to authenticate `joe` user by token too and configure authorization permissions using his user id. Is it clearer now?

Reply:
@g4s8 my point was that multiple users would have the same login:
```yaml
joe:
  login: joejane@gmail.com
jane:
  login: joejane@gmail.com
```

what happens there? Is that valid?

Reply:
@g4s8 ping

Reply:
@victornoel @olenagerasimova no, it's not valid, we have to validate it somehow. Or decide it later and implement only simple login for now where user id is equal to login user name.

Reply:
@olenagerasimova @g4s8 IMHO it is simpler to reify username and login for now so that there are no validation to do since it will be correct by definition because of the yaml syntax rules. And later we can decide if there is an actual need for a secondary login.